packages/snowpack/src/logger.ts

```diff
@@ -0,0 +1,39 @@
+import * as colors from 'kleur/colors';
+import pino from 'pino';
+
+const LEVEL = {
+  TRACE: 10,
+  DEBUG: 20,
+  INFO: 30,
+  WARN: 40,
+  ERROR: 50,
+  FATAL: 60,
+};
+
+const BRACKET_NAME_MATCH = /^\[[^\]]+\]/; // match [name] in brackets at beginning of string
+
+/** http://getpino.io/#/docs/pretty */
+function prettifier() {
```

Comment:
Since we’re piping all logs through a utility, this is where we can do some consistent formatting.